packages/OS400/curl.inc.in

```diff
@@ -1679,7 +1679,7 @@
      d                 c                   20
       *
      d CURLMIMEOPT_FORMESCAPE...
-     d                                     X'0000000'
+     d                 c                   X'00000000'
```

Comment:
Should be `X'00000001'` !
Thanks for fixing this.

Reply:
Fixed!

Reply:
Thanks!